internal/resources-validation.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+jobs:
+# Spin up a Linux VM for validating code.
+- job: png_resources_validation
+  displayName: PNG Resources Validation
+  pool:
+    vmImage: 'ubuntu-latest'
+
+  steps:
+    - script: |
+        sudo apt update
+        sudo apt install -y pngcrush
+      displayName: 'Install Dependencies'
+    - script: |
+        # Print the full paths of PNG files
+        find . -type f -name "*.png" -print
+        # Run pngcrush command on PNG files and check for Profile issues
+        for file in $(find . -type f -name "*.png"); do
+          output=$(pngcrush -n -q "$file" 2>&1)
+          echo "$output"
+          if [[ "$output" == *"pngcrush: iCCP:"* ]]; then
```

Comment:
I am not crazy about the string parsing approach. Because if a new version of pngcrush change the string, our test will never match again and we won't be aware of it ever since all tests will always pass.

Reply:
Unfortunately, pngcrush currently does not provide a more robust way to check for specific types of errors. It does not return a non-zero exit code when it encounters warnings or non-fatal errors.
To work around this issue, I've pinned pngcrush to a specific version (1.8.13) to ensure that this error strings remain consistent.